src/main/resources/application.properties

```diff
@@ -25,7 +25,7 @@ oauth.clientId=970791974390-1581mjhtp2b3jmg4avhor1vabs13b7ur.apps.googleusercont
 oauth.schemeName=googleoauth
 oauth.loginEndpoint=https://accounts.google.com/o/oauth2/auth
 oauth.scopes=openid,email,profile
-datarepo.resourceId=broad-jade-dev
+datarepo.resourceId=${GOOGLE_CLOUD_PROJECT}
```

Comment:
This is the name of the SAM resource for Data Repo, which is not necessarily the same as the Google project name -- though they may have the same value in practice. Perhaps we can move this variable to helm, similar to the sam.basePath property above?

Reply:
After discussion with Mariko, here is the suggested path forward:

1. Short term, I will revert this change and move the variable to helm.
2. Long term, the team will discuss standardizing it across environments and setting a default in `application.properties`
3. Create helm file overrides in each environment as required